modules/cnf-topology-aware-lifecycle-manager-backup-feature.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+// Epic CNF-3901 (CNF-2133) (4.11), Story TELCODOCS-339
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="talo-backup-start_and_update_{context}"]
+= Creating a ClusterGroupUpgrade CR with backup
+
+For SNO (single node cluster), you can create a backup of a deployment before an upgrade. If the upgrade fails you can use the backup to return the system to its original state.
```

Comment:
return the system to its **original state** -> return the system to its pre-upgrade state

Reply:
corrected